packages/parser/test/specs/deep-circular/deep-circular.test.ts

```diff
@@ -37,23 +40,7 @@ describe('API with deeply-nested circular $refs', () => {
   });
 
   it('should validate successfully', async () => {
-    const api = await validate<ValidAPIDefinition>(relativePath('specs/deep-circular/deep-circular.yaml'));
-
-    expect(api).toStrictEqual(dereferencedAPI);
-    expect(api.paths['/family-tree'].get.responses['200'].schema.properties.name.type).toStrictEqual(
-      api.paths['/family-tree'].get.responses['200'].schema.properties.level1.properties.name.type,
-    );
-    expect(api.paths['/family-tree'].get.responses['200'].schema.properties.name.type).toStrictEqual(
-      api.paths['/family-tree'].get.responses['200'].schema.properties.level1.properties.level2.properties.name.type,
-    );
-    expect(api.paths['/family-tree'].get.responses['200'].schema.properties.name.type).toStrictEqual(
-      api.paths['/family-tree'].get.responses['200'].schema.properties.level1.properties.level2.properties.level3
-        .properties.name.type,
-    );
-    expect(api.paths['/family-tree'].get.responses['200'].schema.properties.name.type).toStrictEqual(
-      api.paths['/family-tree'].get.responses['200'].schema.properties.level1.properties.level2.properties.level3
-        .properties.level4.properties.name.type,
-    );
```

Comment:
this coverage looks useful, could we keep it?

Reply:
`validate` no longer returns an API schema so i can't do these assertions anymore

Reply:
oh, duh